src/lib/ptree/support/snabb-softwire-v2.lua

```diff
@@ -100,13 +101,20 @@ local function remove_softwire_entry_actions(app_graph, path)
    return {{'call_app_method_with_blob', args}, {'commit', {}}}
 end
 
+-- Configuration discontinuity-time: this is set on startup and whenever the
+-- configuration changes.
+local discontinuity_time = os.time()
```

Comment:
Humm, not so great for this to be global.  OK but it's possible to run multiple lwaftr apps in a process.

Reply:
Guess we could wrap the support plug-in in a closure, the `binding_table_instance` variable is currently global as well.